R/steps.R

```diff
@@ -185,10 +185,11 @@ Cicerone <- R6::R6Class(
 #' @param session A valid Shiny session if `NULL` the function
 #' attempts to get the session with [shiny::getDefaultReactiveDomain()].
     start = function(step = 1, session = NULL){
-      
+      # If private$runs isn't initialized this will be TRUE, IE run_once = FALSE will run contained code
       if (private$runs %||% 0 < 1) {
```

Comment:
@JohnCoene These conditionals will make it run only once if `run_once` is TRUE

Reply:
All variables must be initialised within an R6 Class. `public` and `private` are environments that are locked.

This does not run on my end.

Reply:
My bad @JohnCoene , I forgot about that default environment locking in R6 classes. I always leave the environments unlocked in the R6 classes I make so I was operating under false assumptions! be4f26a initializes the `runs` in `private` now.